tests/kernel/mem_protect/userspace/src/main.c

```diff
@@ -23,7 +23,7 @@
 #endif
 
 #if defined(CONFIG_ARM)
-#include <arch/arm/cortex_m/mpu/arm_core_mpu_dev.h>
+extern void arm_core_mpu_disable(void);
```

Comment:
So this made me look at the actual test, why do we have a disable_mmu_mpu() if we don't have an exposed API related to it?

Reply:
@andrewboie FYI

Reply:
@galak I get the feeling that this test is there to show that we can't disable the MPU from user mode, right @andrewboie ?

Reply:
I wouldn't object if we deleted that test. There's some other dubious cases in there as well.

Reply:
I like that it "tests" that user-mode can't play with the mpu :)

Reply:
Are we trying to prove whether the CPU works or the software?
Proving that the CPU works is a waste of time IMO

Reply:
Yeap, that's in fact, correct